test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +35,39 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception {
```

Comment:
This is a correctly written test. 
Please also be aware of the 3 part test name format that may be useful when you write tests for your project.